chat/src/main/java/io/skygear/plugins/chat/ChatContainer.java

```diff
@@ -659,9 +661,10 @@ public void onSucc(@Nullable List<Message> messages) {
                                 mMap.put(m.getId(), m);
                             }
                             for (UserConversation uc: userConversations) {
-                               if (uc.getLastReadMessageId() != null) {
-                                   uc.lastMessage = mMap.get(uc.getLastReadMessageId());
-                               }
+                                Conversation c = uc.conversation;
+                                if (c.getLastMessageId() != null) {
+                                    uc.lastMessage = mMap.get(c.getLastMessageId());
+                                }
```

Comment:
@rickmak Do we still need to fetch the last **read** message of the user conversation ?